Tasks/codepush-release-cordova/task.json

```diff
@@ -0,0 +1,173 @@
+            "helpMarkDown": "HockeyApp service endpoint that is configured with your account credentials.",
+            "visibleRule": "authType = ServiceEndpointHockeyApp"
+        },
+        {
+            "name": "appName",
+            "type": "string",
+            "label": "App Name",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Name of the app you want to release the update for."
+        },
+        {
+            "name": "deploymentName",
+            "type": "pickList",
+            "label": "Deployment",
+            "defaultValue": "Staging",
+            "required": false,
+            "helpMarkDown": "Name of the deployment you want to release the update to.",
+            "options": {
+                "Production": "Production",
+                "Staging": "Staging"
+            },
+            "properties": {
+                "EditableOptions": "True"
+            }
+        },
+        {
+            "name": "platform",
+            "type": "pickList",
+            "label": "Platform",
+            "defaultValue": "android",
+            "required": true,
+            "helpMarkDown": "Platform to build/prepare the release for.",
+            "options": {
+                "android": "Android",
+                "ios": "iOS"
+            }
+        },
+        {
+            "name": "shouldBuild",
+            "type": "boolean",
+            "label": "Build?",
+            "defaultValue": false,
+            "required": true,
+            "helpMarkDown":"This task runs cordova prepare to set up assets by default. Checking this box will tell it to do a cordova build instead of a cordova prepare. Use this if you have a build hook that sets up assets."
+        },
+        {
+            "name": "rollout",
+            "type": "string",
+            "label": "Rollout",
+            "defaultValue": "100%",
+            "required": false,
+            "groupName": "advanced",
+            "helpMarkDown": "Percentage of users this release should be immediately available to."
+        },
+        {
+            "name": "description",
+            "type": "string",
+            "label": "Description",
+            "defaultValue": "",
+            "required": false,
+            "groupName": "advanced",
+            "helpMarkDown": "Description of the changes made to the app in this release.<br />Note: This can be set to **$(Release.ReleaseDescription)** if being run within a release definition, and you want to inherit the release's description."
+        },
+        {
+            "name": "appStoreVersion",
+            "type": "pickList",
+            "label": "Target Binary Version",
+            "defaultValue": "autodetect",
+            "required": false,
+            "groupName": "advanced",
+            "helpMarkDown": "Semver expression that specifies the binary app version(s) this release is targetting (e.g. 1.1.0, ~1.2.3). Default will auto-detect from config.xml.",
```

Comment:
Can we change the last sentence to `Selecting "Auto-Detect" will use the version from your app's config.xml."?